### PR TITLE
New Template - AD credential delivery

### PR DESCRIPTION
**Pull request by arvilnagpal-okta**
@charleschan-okta @uvartak-okta plz review

### CODE REVIEW

File: workflows/send_mgr_ad_credentials/workflow.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+   "name": "send_mgr_ad_credentials",
+   "title": "Send Active Directory Credentials to Manager",
+   "description": "Many organizations use Microsoft Active Directory to manage user credentials, also known as AD DelAuth. While Okta’s Active Directory integration allows for user provisioning, organizations need a solution to communicate the account credentials to the user. When onboarding new hires, companies may need to set up these accounts ahead of time. However, the user may not have system or email access until the day of joining. In these scenarios, companies can email the account credentials to the user’s manager with a one-time password. This template helps demonstrate how to identify users being added to Active Directory using Okta’s “User Assigned to Application” event, fetch their manager’s email address and send an email notification.",
+   "connectors": [
+      "okta"
```
Comment (charleschan-okta) [inline workflows/send_mgr_ad_credentials/workflow.json:6]: missing office365mail connector, which is used in the flow

**Review by charleschan-okta — CHANGES_REQUESTED** [review]

File: workflows/send_mgr_ad_credentials/workflow.json (hunk 1 above)
Comment (arvilnagpal-okta) [reply workflows/send_mgr_ad_credentials/workflow.json:6]: good catch

Comment (arvilnagpal-okta) [reply workflows/send_mgr_ad_credentials/workflow.json]: done

**Review by charleschan-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**